README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is in `stripe` using `ACH`.
```

Comment:
Is `ACH` is an enum constant?
Either:
- clarify the enum to which you refer,
- style as an initialism: ACH